app/routers/event_images.py

```diff
@@ -0,0 +1,216 @@
+from functools import lru_cache
+from nltk.tokenize import word_tokenize
+from typing import Optional
+from word_forms.lemmatizer import lemmatize
+
+import re
+
+
+FLAIRS_EXTENSION = '.jpg'
+FLAIRS_REL_PATH = r'..\static\event_flairs'
```

Comment:
Awesome progress, nearly done in my opinion.

It might be a bit dangerous to use `..\static` in your code.
If we change the project structure, the code will break and the tests won't find it.

Can you somehow make the `static`  directory configurable?
Maybe we should put it in `config` and make it absolute using `os.path.absolute`

Reply:
Done!